index.html

```diff
@@ -15,18 +15,23 @@
   <body>
     <header class="header">
       <a href="#headline-section" class="header-logo">My Logo</a>
-      <nav>
-        <button 
-          class="only-mobile hamburger-btn"
+      <nav class="navbar">
+        <!-- <button 
+          class="only-mobile hamburger-btn hamburger"
           data-mobile-menu-target="#mobile-menu"
         >
           <img src="Union.svg" alt="Hamburger Navbar Icon" />
-        </button>
-        <ul class="only-desktop menu-bar">
-          <li><a href="#work-section">Portfolio</a></li>
-          <li><a href="#about-section">About</a></li>
-          <li><a href="#contact-form">Contact</a></li>
+        </button> -->
+        <ul class="only-desktop menu-bar nav-menu">
+          <li class="nav-item"><a class="nav-link" href="#work-section">Portfolio</a></li>
+          <li class="nav-item"><a class="nav-link" href="#about-section">About</a></li>
+          <li class="nav-item"><a class="nav-link" href="#contact-form">Contact</a></li>
         </ul>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
       </nav>
```

Comment:
- [ ] Great job implementing the mobile menu :clap: / I have realized that there is a white space at the top of the menu when it opens. Kindly fix this to make your mobile menu perfect!

**This is how I am seeing it on my end:**

![image](https://user-images.githubusercontent.com/86318284/188631380-79769e40-4d5f-4ccf-8143-99c4ede0bd39.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/188631495-2db8af72-6a44-473d-8413-b781a1cfd3e8.png)